composer.json

```diff
@@ -21,14 +21,20 @@
         "tecnickcom/tcpdf": "^6.2.22"
     },
     "require-dev": {
-        "atoum/atoum": "^3.1",
+        "phpunit/phpunit": "^5.7.27|^6",
         "friendsofphp/php-cs-fixer": "^2.16.3"
     },
     "autoload": {
         "psr-0": {
             "Smalot\\PdfParser\\": "src/"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "Test\\Smalot\\PdfParser\\": "test/",
+            "Tests\\Smalot\\PdfParser\\": "tests/"
```

Comment:
Why do you have `test` & `tests`?
Keep only `tests`, it's enough.

Reply:
`test` contains test related classes, like `TestCase` or maybe traits etc. later on. Its the place for other helper classes too.

`tests` only contains the tests itself.

I saw this separation on a few projects and find it easier to maintain this way.

Reply:
Any objections here?